xapian-applications/omega/templates/query

```diff
@@ -52,50 +52,55 @@ document.write("<span>Modified:<time datetime=\""+D+" "+T+"\"><b>"+D+"</b></time
 <body>
 <form name=P method=get
 action="$html{$or{$env{SCRIPT_NAME},omega}}" target="_top">
+<div class="query-container">
 ${Search Control(Start) - shows search button, options and textbox}
-<input id="omega-autofocus" type=search name=P value="$html{$query}" size=65 autofocus>
+<input class="query-textbox" id="omega-autofocus" type=search name=P value="$html{$query}" size=65 autofocus>
 <script>
 if (!("autofocus" in document.createElement("input")))
  document.getElementById("omega-autofocus").focus();
 </script>
-<input type=submit value="Search">
+<input class="query-search-button" type=submit value="Search">
 ${Todo: Replace radios with search buttons}
 $if{,$include{inc/anyalldropbox},
-<fieldset>$include{inc/anyallradio}</fieldset>}
+<fieldset class="query-options">$include{inc/anyallradio}</fieldset>}
+</div>
 ${Search Control(End)}
 ${Suggestions(Start) - shows suggested terms to improve query}
 $if{$opt{topterms},
- <div title="Suggested terms to add to your query"
+ <div class="query-suggested-container" title="Suggested terms to add to your query"
  style="text-align:left;background:#cfc;border:1px solid green;padding:2px;font:11px verdana$. arial$. helvetica$. sans-serif;">
- $map{$topterms,<span style="white-space:nowrap"><input type=checkbox name=X value="$html{$prettyterm{$_}}" onClick="C(this)">$html{$prettyterm{$_}}</span> }
+ $map{$topterms,<span class="query-suggested-element" style="white-space:nowrap"><input type=checkbox name=X value="$html{$prettyterm{$_}}" onClick="C(this)">$html{$prettyterm{$_}}</span> }
  <noscript><input type=hidden name=ADD value=1></noscript>
  </div>
 }
-$if{$suggestion,Did you mean: <a href="$html{$or{$env{SCRIPT_NAME},omega}?DB=$url{$dbname}&P=$url{$suggestion}$if{$ne{$hitsperpage,10},&HITSPERPAGE=$hitsperpage}$if{$fmt,&FMT=$url{$fmt}}$if{$cgi{COLLAPSE},&COLLAPSE=$url{$cgi{COLLAPSE}}}$map{$relevants,&R=$_}$if{$cgi{THRESHOLD},&THRESHOLD=$url{$cgi{THRESHOLD}}}$map{$cgilist{B},&B=$url{$_}}&SPELL=0}">$html{$suggestion}</a>}
+$if{$suggestion,Did you mean: <a class="query-suggestion" href="$html{$or{$env{SCRIPT_NAME},omega}?DB=$url{$dbname}&P=$url{$suggestion}$if{$ne{$hitsperpage,10},&HITSPERPAGE=$hitsperpage}$if{$fmt,&FMT=$url{$fmt}}$if{$cgi{COLLAPSE},&COLLAPSE=$url{$cgi{COLLAPSE}}}$map{$relevants,&R=$_}$if{$cgi{THRESHOLD},&THRESHOLD=$url{$cgi{THRESHOLD}}}$map{$cgilist{B},&B=$url{$_}}&SPELL=0}">$html{$suggestion}</a>}
 ${Suggestions(End)}
 ${Query metadata(start) - shows query performance or document size}
 $or{$html{$error},
+<div class="query-metadata">
```

Comment:
`div` isn't semantic, and should be avoided if possible. What does the text look like here? Something like the following:

> Searching 71774 documents, 11-20 of about 200 matches

Then a break, and the query terms and term frequencies, then the search time. That's all close to narrative content, so just wrap it in `p`, which is otherwise identical to `div`.